paddle/framework/var_desc.cc

```diff
@@ -18,6 +18,10 @@ limitations under the License. */
 namespace paddle {
 namespace framework {
 
+VarDesc::VarType VarDescBind::GetType() const { return desc_.type(); }
```

Comment:
I see that we added methods `GetType`  and `SetType`, but I don't see them being used anyway in this PR. Just a kindly reminder -- do we really need them?

Reply:
I just modify the syntax of these two methods.
Before, the methods implementation are also in header file. I move the implementation to source file to take consistency with other methods in class VarDescBind.